### DISCUSSION TITLE
Bravo saying Not Available

### DISCUSSION THREAD

**Discussion by LesegoT** [sql-bi/Bravo#669] [Q&A] [answered]
Hello! Since updating my Power BI Desktop to version 2.117.984.0 (May 2023) my Bravo is showing Not Available in the top right corner. It starts up okay and analyses the model, but I am unable to format DAX or Manage Dates. Nothing else has changed in my environment (same network settings, using System proxy setting). I have uninstalled and re-installed and tried it with different pbix file with no luck, even with files I previously used Bravo to create the Dates table in. What could be the problem?

![Screenshot 2023-05-31 085208](https://github.com/sql-bi/Bravo/assets/30831601/86c2556b-b030-46b9-bd24-c3d46f8643e4)

**Comment by albertospelta** [comment, marked as answer]
Hi @LesegoT, I created an new issue here https://github.com/sql-bi/Bravo/issues/671 since it's probably a bug.
